plasmapy/analysis/bdot_probe.py

```diff
@@ -0,0 +1,31 @@
+"""_summary_
```

Comment:
Remove this file from this pull request (PR).  We'll include this in its own PR.